armi/bookkeeping/db/databaseInterface.py

```diff
@@ -145,7 +145,12 @@ def interactEveryNode(self, cycle, node):
 
         DBs should receive the state information of the run at each node.
         """
-        # skip writing for last burn step since it will be written at interact EOC
+        if self.o.cs["numCoupledIterations"]:
```

Comment:
add comment saying that h5 cant handle writing more than once so we only write at the end

Reply:
Addressed https://github.com/terrapower/armi/pull/1005/commits/661e26a9a2a148719549ff717ce4fd0b745b2ef8